src/lib/handlers/db.js

```diff
@@ -33,3 +39,26 @@ const createUserDocument = async (user) => {
 	console.log('User created');
 	return;
 };
+
+export const deleteTrick = async (trickId) => {
+	try {
+		setLoading(true)
+		const userRef = doc(db, 'users', user.uid);
+		const tricksCollection = collection(userRef, 'tricks');
```

Comment:
This is a repetition of the code in the main layout. We should create a function to get trick collection of a user